db/migrations/20210113172906_add_circles_to_draft.js

```diff
@@ -0,0 +1,13 @@
+const table = 'draft'
+
+exports.up = async (knex) => {
+  await knex.schema.table(table, function (t) {
+    t.specificType('circles', 'text ARRAY')
```

Comment:
I think the product logic we have now is one draft can only have one circle. This is quite unlikely to change in the future. Do we still need to use array here for future proof?